build.sbt

```diff
@@ -634,8 +633,6 @@ lazy val unit = project
       "ch.epfl.scala" %% "bloop-config" % V.bloop,
       "org.scalameta" %% "munit" % V.munit,
       // The dependencies listed below are only listed so Scala Steward
-      // will pick them up and update them. They aren't actually used.
-      "com.lihaoyi" %% "ammonite-util" % V.ammonite intransitive (),
```

Comment:
I just removed this... util is no longer published for 12, so this won't work with the new version. We also can't put the repl-api here which is what would be accurate because it needs to be exact. I was annoyed a bit by this mind gymnastics of figuring out the right thing to put here, so I think we should just remove it. Each month or so if we do a new release we always look to see if the new Ammonite is published for that version anyways, so we can just update it that way.